config/autoload.php

```diff
@@ -12,7 +12,10 @@
 /**
  * Register namespace
  */
-NamespaceClassLoader::add('Rhyme', 'system/modules/isotope_feeds/library');
+if (class_exists('NamespaceClassLoader'))
+{
+    NamespaceClassLoader::add('Rhyme', 'system/modules/isotope_feeds/library');
```

Comment:
It's a fallback for those not using composer. They can install the NamespaceClassLoader and will be able to use this extension too.